sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Release History
-## 1.2.0-beta.1 (Unreleased)
-
-### Features Added
+## 2.0.0 (2022-08-26)
+### Bugs Fixed
+- Push notification registration renewal not guaranteed to happen when Contoso APP is idle or closed.
 
 ### Breaking Changes
-
-### Bugs Fixed
+- Introduced WorkerManager library which breaks push notification for existing Contoso APPs. (Check https://docs.microsoft.com/azure/communication-services/tutorials/chat-android-push-notification)
 
 ### Other Changes
+- Implement internal key management library to securely persist all the registration secret keys which are not expired.
+- Increase minSDKVersion from 21 to 23.
```

Comment:
Not sure if this should be under the breaking changes section, as it can technically be "reversed". Any thoughts @JonathanGiles?

Reply:
I think "Increase minSDKVersion from 21 to 23" could be put under breaking changes. Because it is breaking change in this release (2.0.0). When we do a reverse, it would be done in another release like 2.1.0. 

Additionally, put it under breaking change section is more obvious for users. They are likely to ignore the "Other changes" section.

Reply:
Sounds good, let's do it then.